### DISCUSSION TITLE
Editing Webpage

### DISCUSSION THREAD

**Discussion by Jellybeanz1980** [UtilitechAS/amsreader-firmware#597] [Q&A]
Hello,

I'd like to edit the main webpage for showing a graph for comparison. I try to edit .svelte files, but when I compile the project, the modification doesn't seem to be included. Could you point me to the right direction for editing the webpages?

Best regards,
Flemming

**Comment by Jellybeanz1980** [comment]
Never mind, I found out how :)

**Reply by MichaelUray** [reply]
How did you solve it?
I guess you have to build Svelte somehow.
